jme3-android/src/main/java/com/jme3/texture/plugins/AndroidBufferImageLoader.java

```diff
@@ -158,8 +153,7 @@ public Object load(AssetInfo assetInfo) throws IOException {
         data.flip();
         
         bitmap.recycle();
-        
-        Image image = new Image(format, width, height, data, ColorSpace.sRGB);
-        return image;
+       
+        return new Image(format, width, height, data, ColorSpace.sRGB);
```

Comment:
I prefer the old code, which assigned the new object to a variable and returned the variable. That made it easy to examine the new object in the debugger.

Reply:
Changes Applied